sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ChangeFeedProcessorBuilder.java

```diff
@@ -98,6 +103,35 @@ public ChangeFeedProcessorBuilder handleChanges(Consumer<List<JsonNode>> consume
         return this;
     }
 
+    /**
+     * Sets a consumer function which will be called to process changes for full fidelity
```

Comment:
nit: should we also update the full fidelity mentioned in the comments to the new name?